src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -171,10 +171,13 @@ public void addTag(Tag t) throws UniqueTagList.DuplicateTagException {
      * @throws TagNotFoundException
      * @throws IllegalValueException
      */
-    public void removeTag(String tagGettingRemoved) throws TagNotFoundException, IllegalValueException {
+    public void removeTag(String tagGettingRemoved) throws TagNotFoundException,
+            IllegalValueException, PersonNotFoundException {
         Tag tagToRemove = tags.removeTag(tagGettingRemoved);
         for (Person person: persons) {
+            Person temp = person;
```

Comment:
If you are directly accessing the `persons` list in the address book, `removeTag` should affect the underlying list directly - no need for the temp variable.

Reply:
yup it affects the underlying list directly, but the changes are not immediately reflected on the console.

Reply:
Noted!